### ISSUE TITLE
home= kwarg in Character.create method doesn't work.

### ISSUE THREAD

**Issue by kovitikus** [evennia/evennia#1921]
#### Brief summary of issue / Description of requested feature:
Was trying to make a custom room for a new account to start in and in the `Account.create` method, it calls upon `Character.create`. I made the room just before this call and passed the new room as the home kwarg. It did not work properly and instead the character is created with the default `Limbo (#2)` room. Once the new account is made, both the room and character exist.

Original section of the Account.create method I'm attempting to change.
https://github.com/evennia/evennia/blob/master/evennia/accounts/accounts.py#L732

My custom code that doesn't work. 
(Related section on **line 150**)
https://pastebin.com/fPFnNuj1

#### Steps to reproduce the issue / Reasons for adding feature:
Call the Character.create method and pass it a home kwarg to use.
```
!from typeclasses.characters import Character; Character.create("Batman", "Kovitikus", home="#212")
```

#### Error output / Expected result of feature
When calling the `Character.create` method via `@py`, no resulting character is made.
https://i.imgur.com/QlZCm5M.png

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:
Evennia master branch.

**Comment by strikaco** [comment]
(On the subject of creation kwargs, the `description` kwarg also does not work as expected; I think it might need to be shortened to `desc` to be apply.)

The reason for this issue appears to be this line:

https://github.com/evennia/evennia/blob/c6aa94365bdb14aa94489f8b8e756234c58f7808/evennia/objects/objects.py#L1968

It expects a dbid, not an object. Try passing the id instead of the object as the kwarg value.

**Comment by kovitikus** [comment]
In my @py command, I did try passing the dbid instead and it never created a character, as you can see in the screenshot with the objects listed after I executed the @py command. 

Also, I was told by Griatch that passing the object to the Character.create method should work as well. Perhaps that isn't true. `settings.DEFAULT_HOME` does use the dbid afterall.

Thanks for the response!

**Comment by strikaco** [comment]
Sorry, missed the link for the screenshot.

Pretty sure the `utils.create` method will accept objs or ids; the `Character.create` method definitely only accepts an id. Most of the code for the latter was ganked from a Command, so the input parameter types were limited to that which a user could type into a terminal-- i.e. strings.

That is odd though!

**Comment by strikaco** [comment]
This turned out to be two problems.
* You do have a syntax error in your create statement-- the second arg should be an Account object or None, not a string (I didn't recognize it since I rarely use that arg). But that also surfaced an actual bug:
* You managed to hit a line I wrote that assumed an account object was always provided as the second arg, or defaults to None. The value you were providing was neither, so the creation was failing when you hit it and the errors were being swallowed by something higher in the stack.

**Comment by strikaco** [comment]
This should be closed by #2002; would you kindly confirm?

**Comment by Griatch** [comment]
Closing this since it seems to be addressed in #2002. @kovitikus - if there are any remaining issues for your implementation that was missed, note it here and we can re-open.
